src/toplevel_generators.jl

```diff
@@ -158,13 +163,17 @@ function generate_wrapper_load(src_name, pkg_uuid, __source__)
                 end
 
                 # From the available options, choose the best wrapper script
-                select_platform(valid_wrappers)
+                if host_platform !== nothing
```

Comment:
Maybe add a comment to explain why this is necessary, to avoid further confusion in the future?

Reply:
I kinda already forgotten the reason.

Reply:
Thank your past self who wrote this message: https://github.com/JuliaPackaging/JLLWrappers.jl/pull/35#issuecomment-999051946 😉

Reply:
```suggestion
                # The two argument `select_platform` is notably slower, so micro-optimize this by
                # only calling it when necessary.
                if host_platform !== nothing
```